mdata/aggmetric.go

```diff
@@ -148,8 +148,8 @@ func (a *AggMetric) SyncAggregatedChunkSaveState(ts uint32, consolidator consoli
 
 func (a *AggMetric) GetAggregated(consolidator consolidation.Consolidator, aggSpan, from, to uint32) (Result, error) {
 	// no lock needed cause aggregators don't change at runtime
-	for _, a := range a.aggregators {
-		if a.span == aggSpan {
+	for _, aggregator := range a.aggregators {
```

Comment:
This rename change is good (removes shadowing of `a`) but should have been in a separate commit.

Reply:
The first commit could be just the variable rename. No worries though